package.json

```diff
@@ -69,7 +71,7 @@
       "git add"
     ],
     "*.md": [
-      "remark -qfo",
+      "remark -qf",
```

Comment:
Just curious, why did you remove the `-o`? If I understand correctly, that's similar to ESLint's `--fix`, it fixes what it can so in cases where all errors of a file are auto-fixable, it proceeds to `git add` and you don't have to do anything.

Reply:
Apparently it didn’t work with the `-o`. I’m not sure exactly what lint-staged is doing, but I believe it’s outputting it somehow anyway.

Reply:
👍